Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
Nit: Place at top of file, before usings

Reply:
Oh wow, sorry.